src/main/java/cz/upce/fei/inptp/zz/entity/TimeSlot.java

```diff
@@ -3,37 +3,63 @@
 import java.util.Objects;
 
 public class TimeSlot {
-    
+
     private Day day;
-    private int hour;
-    
-    // TODO: is this clear?
-    private int duration;
-    
-    public TimeSlot(Day day, int hour, int duration) {
+    private static final int MIN_HOUR = 8;
+    private static final int MAX_HOUR = 20;
+    private int startHour = MIN_HOUR;
+    private int endHour = MAX_HOUR;
```

Comment:
From startHour, endHour is not clear if both limits are considered as inclusive, exclusive or some combination...